app/views/idv/verify_errors/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render(
+      'idv/shared/error',
+      title: t('titles.failure.information_not_verified'),
+      heading: t('idv.failure.verify.heading'),
+    ) do %>
+    <p>
+      <% if decorated_session.sp_name.present? %>
+        <%= link_to(
+              t('idv.failure.verify.fail_link_html', sp_name: decorated_session.sp_name),
+              return_to_sp_failure_to_proof_path(
+                step: 'verify_info',
+                location: request.params[:action],
+              ),
+              class: 'usa-link--external',
+            ) %>
+      <% else %>
+        <%= link_to(
+              t('idv.failure.verify.fail_link_html', sp_name: APP_NAME),
+              account_path,
+              class: 'usa-link--external',
+            ) %>
```

Comment:
We typically only use external links in combination with opening a new tab, which we're not doing here. I _think_ a case could be made to include it here, since it is technically an external site, though we also link back to the SP in many other places without it. The second link here is definitely not external though.

For the first link, I'd suggest either (a) removing the external styling since it opens in the same tab or (b) make the link open in a new tab.

```suggestion
        <%= link_to(
              t('idv.failure.verify.fail_link_html', sp_name: decorated_session.sp_name),
              return_to_sp_failure_to_proof_path(
                step: 'verify_info',
                location: request.params[:action],
              ),
            ) %>
      <% else %>
        <%= link_to(
              t('idv.failure.verify.fail_link_html', sp_name: APP_NAME),
              account_path,
            ) %>
```